package.json

```diff
@@ -21,7 +21,12 @@
     "analyze": "npm run clean && npm run build:prod -- --stats-json && webpack-bundle-analyzer ./dist/stats.json",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate && node .all-contributors-html.js",
-    "contributors:check": "all-contributors check"
+    "contributors:check": "all-contributors check",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "precy:ci": "npm run build",
+    "cy:ci": "npm-run-all --parallel --race start:ci \"cy:run -- --config=baseUrl=http://localhost:4000\"",
+    "start:ci": "angular-http-server --path ./dist -p 4000 --silent"
```

Comment:
Could we prefix everything with `cy:` so that it is obvious that they belong together?

Reply:
Yes!